subsys/bluetooth/mesh/net.h

```diff
@@ -88,7 +88,7 @@ struct bt_mesh_friend {
 		uint32_t start;                  /* Clear Procedure start */
 		uint16_t frnd;                   /* Previous Friend's address */
 		uint16_t repeat_sec;             /* Repeat timeout in seconds */
-		struct k_delayed_work timer;  /* Repeat timer */
+		struct k_work_delayable timer;   /* Repeat timer */
```

Comment:
Looks like a space added before a tab here, though if checkpatch doesn't complain it doesn't need to be fixed.

Reply:
I don't believe so. The comment alignment is added manually.

Reply:
OK; then it's probably aligned with spaces, with one added.  The codebase isn't consistent on spaces or tabs for internal alignment whitespace on lines.

Reply:
The general approach we've tried to follow for Bluetooth code is "tabs for indentation, spaces for alignment", or in other words "tabs should only exist in the beginning of a line but never somewhere in the middle". The reason for that is that this tends to guarantee that when viewed e.g. in the GitHub web UI everything will look correctly aligned. I *think* it should also help viewing in general regardless of what tab-width people have configured in their editor, but this I'm not 100% sure of. Anyway, I haven't cared enough to propose having this specified in some project-wide coding style document, but maybe it would be worth doing it?

Reply:
I know there's significant variation throughout the codebase for alignment of backslashes at the end of multi-line macros (one space, space aligned, tab aligned).  Something to consider if #21392 gets some attention.